### PR TITLE
lxi: add page

### PR DESCRIPTION
**Pull request by peterbabic**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/lxi.md
Comment (CleanMachine1) [inline pages/linux/lxi.md:20]: Maybe make the ip more generic, 192.168.*.100 or 10.0.0.100

Comment (navarroaxel) [reply pages/linux/lxi.md:20]: Usually we use a place holder like `ip_address`, also the `"` should be outside the tokens: `"{{*IDN?}}"`

File: pages/linux/lxi.md
Comment (navarroaxel) [inline pages/linux/lxi.md:22]: I'm not sure about this description :thinking:, maybe:
```suggestion
- Run a benchmark for request and response performance:
```

File: pages/linux/lxi.md
Comment (CleanMachine1) [reply pages/linux/lxi.md]: https://github.com/tldr-pages/tldr/pull/6174/commits/188ab49aa22917135c067941e7f3a5d4ce0fe9ed

File: pages/linux/lxi.md
Comment (CleanMachine1) [inline pages/linux/lxi.md:3]:
```suggestion
> Control `LXI` compatible instruments such as oscilloscopes.
```

Controlling was also spelt wrong

File: pages/linux/lxi.md
Comment (CleanMachine1) [inline pages/linux/lxi.md:18]:
```suggestion
- Send a SCPI command to an instrument:
```

File: pages/linux/lxi.md
Comment (CleanMachine1) [inline pages/linux/lxi.md:6]:
```suggestion
- Discover `LXI` devices on available networks:
```

**Comment by CleanMachine1** [comment]
After these, you have my approval

File: pages/linux/lxi.md
Comment (peterbabic) [reply pages/linux/lxi.md:18]: Hello @CleanMachine1, my pleasure.
Thank you for another set of reviews.

I am not sure about this one, however. Previously I was corrected by @bl-ue that it should be `an SCPI`, because S is spelled as `es`. 

More at https://github.com/tldr-pages/tldr/pull/5215#discussion_r567959276

Is that related?

File: pages/linux/lxi.md
Comment (peterbabic) [reply pages/linux/lxi.md:3]: Is there some new rule for the \` around LXI change, that I've failed to acknowledge, please?

File: pages/linux/lxi.md
Comment (peterbabic) [reply pages/linux/lxi.md:6]: Detto about the usage of \` - is there a justification for such change, please? (to know better in the future)

File: pages/linux/lxi.md
Comment (CleanMachine1) [reply pages/linux/lxi.md:3]: No, but it makes more sense in the fact that it sounds like a standard, if its just a generic software then feel free to just change the other bit suggested

File: pages/linux/lxi.md
Comment (CleanMachine1) [reply pages/linux/lxi.md:18]: `an` is correct, my suggestion has been updated, good spot rather than just accepting

File: pages/linux/lxi.md
Comment (CleanMachine1) [reply pages/linux/lxi.md:6]: See previous review for my reply

**Comment by bl-ue** [comment]
Hi, I was pinged on this page, and I'm not fully participating, but I'd like to point out that `ip_adress` is a typo (should be `ip_address`).

**Comment by CleanMachine1** [comment]
Good spot bl-ue, could you check gitter

File: pages/linux/lxi.md
Comment (sbrl) [reply pages/linux/lxi.md:3]: I'm not sure the backticks make sense here.
```suggestion
> Control LXI compatible instruments such as oscilloscopes.
```

File: pages/linux/lxi.md
Comment (sbrl) [reply pages/linux/lxi.md:6]: No, I'm not sure backticks make sense here either.

**Comment by peterbabic** [comment]
> Hi, I was pinged on this page, and I'm not fully participating, but I'd like to point out that `ip_adress` is a typo (should be `ip_address`).

Sorry for pinging but on the other hand, your keen eye is trained well 😃 

Thanks and fixed!

File: pages/linux/lxi.md
Comment (CleanMachine1) [reply pages/linux/lxi.md:6]: I had a feeling they would be controversial. Feel free to resolve if agreed however leave open for now

File: pages/linux/lxi.md
Comment (aherst) [inline pages/linux/lxi.md:14]: How about:

```suggestion
- Capture a screenshot using the specified plugin:

File: pages/linux/lxi.md
Comment (marchersimon) [inline pages/linux/lxi.md:10]: Is the plugin called `autodetection` or does it autodetect plugins?

Comment (marchersimon) [reply pages/linux/lxi.md:10]: In case it's called `autodetection`, I'd say
```suggestion
- Capture a screenshot using the autodetection plugin:
```

Comment (peterbabic) [reply pages/linux/lxi.md:10]: It does plugin autodetection

File: pages/linux/lxi.md
Comment (peterbabic) [reply pages/linux/lxi.md:14]: I am tempted to accepted it but, could you please consider a shorter version like this?
```suggestion
- Capture a screenshot using a plugin:
```

Comment (CleanMachine1) [reply pages/linux/lxi.md:14]:
```suggestion
- Capture a screenshot using a specified plugin:
```

I prefer a to the

Comment (CleanMachine1) [reply pages/linux/lxi.md:14]: We normally use specific and specified depending on the situation.

Specific may fit better here actually

File: pages/linux/lxi.md
Comment (CleanMachine1) [reply pages/linux/lxi.md:10]: Wait for @marchersimon to get a reply before committing

File: pages/linux/lxi.md
Comment (marchersimon) [inline pages/linux/lxi.md:10]:
```suggestion
- Capture a screenshot, detecting a plugin automatically:
```
I'm not sure if this is correct, but the original reads a bit hard

**Review by marchersimon — APPROVED** [review]
Thanks @peterbabic

**Review by sbrl — APPROVED** [review]
Thanks, @peterbabic!

**Pull request merged**
